airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2863,6 +2863,11 @@ components:
           description: The execution date.
           type: string
           format: datetime
+          deprecated: true
```

Comment:
Given the `include|future` and `include_past` I'm not 100% sure we can deprecate this -- does it "make sense" to pass a run_id and then say `include_past`, or does that make more sense to stick with execution date?

(I honestly don't have an opinion right now)

Reply:
I think the execution_date in this use case is needed. I would go with closing this PR because if we still use run_id, we would still need the execution_date

Reply:
We should support clearing just the tasks for a single dag run by ID.

(We should _look_ at refactoring the chain of methods we call to do the work)

Reply:
I have added support running with either execution_date or run_id